groups/alteryx-live/variables.tf

```diff
@@ -117,8 +117,15 @@ variable "alteryx_server_instance_count" {
 }
 
 variable "alteryx_server_ami" {
```

Comment:
Can we rename this to `alteryx_server_ami_id` so it's in keeping with the description